BDArmory/Modules/ModuleWeapon.cs

```diff
@@ -4148,7 +4148,7 @@ void UpdateTargetVessel()
                     }
                     targetVelocity -= Krakensbane.GetFrameVelocityV3f();
                     targetRadius = 1;
-                    targetAcceleration = MissileTgt != null && MissileTgt.Vessel != null ? MissileTgt.Vessel.acceleration : Vector3.zero;
+                    targetAcceleration = MissileTgt != null && MissileTgt.Vessel != null ? MissileTgt.Vessel.acceleration : Vector3d.zero;
```

Comment:
For some reason, my VS env required this change to compile. Not sure how it made it into the dev branch as is.

Reply:
`targetAcceleration` is a `Vector3`, so the correct fix is to cast `MissileTgt.Vessel.acceleration` to a `Vector3` instead. Curiously, it doesn't cause any build issues for me (with `dotnet` version 5.0.404 on Linux), so either the compiler is figuring it out or it's doing some extra conversions.